servers/mcp-neo4j-cypher/src/mcp_neo4j_cypher/server.py

```diff
@@ -264,6 +268,14 @@ async def main(
             password,
         ),
     )
+    custom_middleware = [
+        Middleware(
+            CORSMiddleware,
+            allow_origins=allow_origins,
+            allow_methods=["*"],
```

Comment:
From the tests it seems that the only method allowed is `POST`. Is that the case? If so it might be worth restricting allow_methods to just `POST`